src/main/java/org/fundacionjala/sfdc/pageobjects/Main.java

```diff
@@ -0,0 +1,34 @@
+package org.fundacionjala.sfdc.pageobjects;
+
+import org.openqa.selenium.WebDriver;
+import java.util.Properties;
+import org.fundacionjala.sfdc.commons.DriverManager;
+import org.fundacionjala.sfdc.commons.PropertiesManager;
+
+/**
+ * Main.java
+ * Class to run project by itself.
+ */
+public final class Main {
+    /**
+     * Constructor.
+     */
+    private Main() {
+    }
+
+    /**
+     * Main method.
+     * @param args from cmd execution.
+     */
+    public static void main(final String[] args) {
+        WebDriver driver;
+        Properties prop;
+
+        driver = DriverManager.getInstance().getNavigator();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this instance-reference to a static reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2209)